src/utils.ts

```diff
@@ -421,7 +421,7 @@ export function parseTsAliases(basePath: string, paths: ts.MapLike<string[]>) {
 
   for (const [pathWithAsterisk, replacements] of Object.entries(paths)) {
     const find = new RegExp(
-      `^${pathWithAsterisk.replace(regexpSymbolRE, '\\$1').replace(asteriskRE, '(.+)')}$`
+      `^${pathWithAsterisk.replace(regexpSymbolRE, '\\$1').replace(asteriskRE, '(?!\\.\\/)([^*]+)')}$`
```

Comment:
Previous pattern (`(.+)`) was matching everything when the `*` pattern was used as a path alias. This matches everything except relative paths.

Reply:
Just realized that this was not going to work for relative parents (`../`) so I've adjusted